extension/lib/display.js

```diff
@@ -23,10 +23,10 @@ class Display {
     - readableDomIds: string[]. List of dom IDs that contain readable content.
     - speed: int. Value proportional to the speed of auto-read mode, before adjustment by sentence length. 
     */
-    constructor(readableDomIds, speed) {
+    constructor(readableDomIds, speed, total_words) {
```

Comment:
pls add comment. i guess this is the total number of tokens / words in the entire document?